recipes/hmmcopy/build.sh

```diff
@@ -22,3 +22,7 @@ cd $PREFIX/bin
 sed -i'.bak' "s,internal/,," generateMap.pl
 sed -i'.bak' "s,../renameChr.pl,renameChr.pl," generateMap.pl
 sed -i'.bak' "s,../bigwig/,," generateMap.pl
+sed -i'.bak' "s,/usr/bin/perl -w,/usr/bin/env perl," generateMap.pl
+sed -i'.bak' "s,/usr/bin/perl -w,/usr/bin/env perl," readToMap.pl
+sed -i'.bak' "s,/usr/bin/perl -w,/usr/bin/env perl," renameChr.pl
+rm *.bak
```

Comment:
What is the difference between removing '.bak' from the sed commands and removing the .bak files after?
If there is none, it might be better to just never generate the '.bak. files.

Reply:
Hi! Sure, I can add a test. The sed command works differently on macOS and linux when you include -i; macOS requires a file extension for a backup file but leaving a zero length extension causes linux to fail. So I did it this way so both the linux and macOS tests pass, and I delete the resulting backup files so they don't stay on PATH (happy to hear if there is a better way to do this).

Reply:
I see, I wasn't aware of the macOS issue. Thanks for explaining.